rsocket-core/src/main/java/io/rsocket/loadbalance/PooledRSocket.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.rsocket.loadbalance;
+
+import io.rsocket.RSocket;
+
+public interface PooledRSocket extends RSocket {
+
+  Stats stats();
+
+  LoadbalanceTarget supplier();
+
+  boolean markForRemoval();
+
+  boolean markActive();
```

Comment:
Add some Javadoc on semantics, what happens or should happen, return values, etc. There is some Javadoc in the  concrete class `ResolvingPooledRSocket`. That could be moved up here and used as starting point.

Reply:
Fixed